tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -0,0 +1,139 @@
+#
+# Next we convert the optical density data to haemoglobin concentration using
+# the modified Beer-Lambert law.
+
+raw_fnirs = mne.preprocessing.beer_lambert_law(raw_od)
+raw_fnirs.plot(n_channels=len(raw_fnirs.ch_names), duration=3000)
+
+
+###############################################################################
+# Remove heart rate from signal
+# -----------------------------
+#
+# The expected response is below 0.5 Hz, and the increase in activity around 1
+# Hz is due to heart rate. So apply a low pass filter to remove unwanted
+# activity.
+
+raw_fnirs.plot_psd(average=True)
+raw_fnirs = raw_fnirs.filter(0.05, 0.7, h_trans_bandwidth=0.2,
+                             l_trans_bandwidth=0.02)
+raw_fnirs.plot_psd(average=True)
+
+###############################################################################
+# Extract epochs
+# --------------
+#
+# The expected response is below 0.5 Hz, and the increase in activity around 1
+# Hz is due to heart rate. So apply a low pass filter to remove unwanted
+# activity.
+
+events, _ = mne.events_from_annotations(raw_fnirs, event_id={'1.0': 1,
+                                                             '2.0': 2,
+                                                             '3.0': 3})
+event_dict = {'Control': 1, 'Tapping/Left': 2, 'Tapping/Right': 3}
+fig = mne.viz.plot_events(events, event_id=event_dict,
+                          sfreq=raw_fnirs.info['sfreq'])
+fig.subplots_adjust(right=0.7)  # make room for the legend
+
+reject_criteria = dict(hbo=80e-6)
+tmin, tmax = -5, 15
+
+epochs = mne.Epochs(raw_fnirs, events, event_id=event_dict,
+                    tmin=tmin, tmax=tmax,
+                    reject=reject_criteria, reject_by_annotation=True,
+                    proj=True, baseline=(None, 0), preload=True,
+                    detrend=None, verbose=True)
+epochs.plot_drop_log()
+
+
+###############################################################################
+# View consistency of responses across trials
+# -------------------------------------------
+#
+# Plot image
+
+epochs['Tapping'].plot_image(combine='mean', vmin=-30, vmax=30)
+epochs['Control'].plot_image(combine='mean', vmin=-30, vmax=30)
+
+
+###############################################################################
+# View consistency of responses across trials
+# -------------------------------------------
+#
+# Plot image
+
+# How do I make all these color maps on the same scale?
+fig, (cont, tapping) = plt.subplots(nrows=2, ncols=2, figsize=(15, 6))
+epochs['Control'].average().plot_image(axes=[cont[0], tapping[0]], show=False)
+epochs['Tapping'].average().plot_image(axes=[cont[1], tapping[1]], show=False)
+fig.show()
+
+
+###############################################################################
+# Plot standard fNIRS response image
+# ----------------------------------
+#
+# Plot image
+
+# This seems a terrible way to do this, I am probably missing something obvious
```

Comment:
Its probably uncommon to plot different types on the same plot? Is there a nicer way to do this?

Reply:
`plot_compare_evokeds` can take a list or dict of evoked objects as its first argument. So put the two conditions in a dict, then call the function just once. Use the `color` parameter (which also can take a dict) to specify which color you want (use the same keys as you used in the dict of evoked objects).

Reply:
Thanks